deploy/kubernetes/rbac.yaml

```diff
@@ -51,6 +51,14 @@ rules:
     resources: ["customresourcedefinitions"]
     verbs: ["create", "list", "watch", "delete", "get", "update"]
 
+# Secret permission is optional.
+# Enable it if your driver needs secret.
+# For example, `csi.storage.k8s.io/snapshotter-secret-name` is set in VolumeSnapshotClass.
+# See https://kubernetes-csi.github.io/docs/secrets-and-credentials.html for more details.
```

Comment:
Comments should be added above lines #35-37 and comment out lines #35-37.

Reply:
updated